src/modules/Reviews/ReviewRepositoryImpl.ts

```diff
@@ -0,0 +1,16 @@
+import Review from './entity/Review'
+import ReviewRepository from './interface/ReviewRepository'
+
+const reviews: Review[] = []
+
+class ReviewRepositoryImpl implements ReviewRepository {
+  async findByUserId(userId: string): Promise<Review[]> {
+    return reviews.filter((review) => review.Id === userId)
+  }
+
+  async create(review: Review): Promise<void> {
```

Comment:
you don't want to return anything for your create review API endpoint I think :D